tests/asr_test.py

```
@@ -0,0 +1,569 @@
+import os
+import time
+from pathlib import Path
+
+import librosa
+import pytest
+import torch
+
+from src.core.asr import ASRChunk, ASRonSPEED
+
+
+def get_gpu_info():
```

Ruff (0.11.9)

Missing return type annotation for public function `get_gpu_info`

(ANN201)